pkg/cmdutil/errors.go

```diff
@@ -44,6 +44,9 @@ func IsUserCancellation(err error) bool {
 	return errors.Is(err, CancelError) || errors.Is(err, terminal.InterruptErr)
 }
 
+// NoChecksError signals that no checks were reported for the pull request
+var NoChecksError = errors.New("NoChecksError")
+
```

Comment:
See the above comment. We don't need this extra error case as "No Checks" isn't an error